### ISSUE TITLE
AWS서버 구축시 navigator.geolocation.getCurrentPosition에 SSL인증이 필요합니다.

### ISSUE THREAD

**Issue by st42597** [SKKU-OSS-Team13/oss-project#6]
[https://stackoverflow.com/questions/39366758/geolocation-without-ssl-connection](url)

getCurrentPosition() and watchPosition() no longer work on insecure origins. To use this feature, you should consider switching your application to a secure origin, such as HTTPS.

일단 깃허브 페이지를 사용해서 테스트를 해보는게 좋을 것 같습니다.

**Comment by PSYcode04** [comment]
네 [github-pages](https://github.com/SKKU-OSS-Team13/SKKU-OSS-Team13.github.io)에 배포하신 것 확인해봤는데 오류가 발생합니다.
![image](https://user-images.githubusercontent.com/71871348/143686968-ac3e0c4c-37f2-4995-9a2d-b8b8876bbb8a.png)

날씨를 불러오기 위해서 요청보내는 부분에서 오류가 발생해서 api 호출을 못하는 것 같아요.
깃 페이지는 정적 웹사이트 호스팅 서비스라서 저희가 생각한 대로 서버에 요청을 보내고 응답을 받은 내용으로 페이지를 다시 렌더링하는 부분을 적용하는 것이 불가능한 것 같습니다.

따라서 `npm start` 명령어로 실행하는 방법으로만 하는 것이 어떨까요? 
서버까지 포함해서 프로젝트를 배포하는 방법을 공부해서 적용하기는 현재는 너무 어렵다고 판단이 됩니다...
